### ISSUE TITLE
#document는 접근을 하지 못하나요?

### ISSUE THREAD

**Issue by TigMonkey** [naver/guitar#414]
특정 사이트를 크롤링 중입니다.

특이한 점은 html 코드 상에 #document라는 것이 있어서 그 하위에 있는 html 태그에 접근을 못하더군요.

그래서 제공해주신 샘플코드와 pdf 파일을 참고하였으나 html 코드 상에 존재하는 #document 에 접근하는 방법에 대해서는 언급이 없었습니다. 

#document 하위에 있는 태그로 접근하는 방법은 없는건가요?

**Comment by ssmmhh** [comment]
브라우저상에 랜더링 되어 있는 태그는 지정가능합니다.  
하지만 JS상으로 동적으로 생성되는 TAG이외의 변수나 값들에 대해서는 접근이 어렵습니다.

**Comment by TigMonkey** [comment]
렌더링이 되어 있는 태그지만
`<iframe>
    <iframe>여기 부분</iframe>
</iframe>`

"여기 부분"으로 표시된 영역의 태그는 못 가져오는 것 같습니다.

**Comment by ssmmhh** [comment]
GUITAR에서 frame하위의 태그들은 접근이 가능하나 iframe 내부의 영역은 여러가지 이슈가 tag 접근이 지원되지 않습니다.

**Comment by TigMonkey** [comment]
네 말씀하신대로 tag 접근이 원할하지 않아 selenium webdriver를 이용하여 예저처럼 xpath로 접근하니 해결됐습니다. 꾸준히 답변 달아주셔서 감사합니다.
